vendor/github.com/cloudbase/garm/params/params.go

```diff
@@ -222,7 +222,9 @@ type BootstrapInstance struct {
 }
 
 type UserDataOptions struct {
-	DisableUpdatesOnBoot bool `json:"disable_updates_on_boot"`
+	DisableUpdatesOnBoot bool     `json:"disable_updates_on_boot"`
+	EnableBootDebug      bool     `json:"enable_boot_debug"`
```

Comment:
This does not seem to be in any commit of garm. We normally don't manually change the contents of the `vendor` folder. It leads to inconsistencies.

But this should be addressed when we split the provider stuff from garm.